src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -85,16 +73,7 @@ public void robotInit() {
 
 		//TODO: Create list of autonomous modes for selector
 		// Setup autonomous mode selectors
-		String[] autoList = {
-				"none",
-				"go"
-		};
-
-		NetworkTable table = NetworkTable.getTable("SmartDashboard");
-		table.putStringArray("Auto List", autoList);
-		LOGGER.debug("Robot Initialized");
```

Comment:
Keep this in so we can deal with deprecated features properly later.

Reply:
Usually I say to take out obsolete code, but for the autonomous code we know we're going to start writing that again too, so it does make sense to leave it in this time.